util/base58/base58.go

```diff
@@ -0,0 +1,172 @@
+/*
+ *
+ * Source: https://github.com/itchyny/base58-go
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2019 itchyny
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ */
+
+package base58
+
+import (
+	"errors"
+	"fmt"
+)
+
+// Errors
+var (
+	ErrorInvalidBase58String = errors.New("invalid base58 string")
+)
+
+// Alphabet: copy from https://en.wikipedia.org/wiki/Base58
+var (
+	//BitcoinAlphabet = NewAlphabet("123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz")
+	//IPFSAlphabet    = NewAlphabet("123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz")
+	//FlickrAlphabet  = NewAlphabet("123456789abcdefghijkmnopqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ")
+	//RippleAlphabet  = NewAlphabet("rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz")
+	TronAlphabet = NewAlphabet("123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz")
+)
```

Comment:
can you just bring `base58`? not sure why we need to have our own implementation

Reply:
Okay, I'll import the package instead.